### PR TITLE
docs(label, list): update usage examples

### PR DESCRIPTION
**Pull request by DitwanP**
**Related Issue:** #6374 

## Summary

Updates component usage examples for the label and list components

### CODE REVIEW

File: packages/calcite-components/src/components/label/usage/Basic.md [hunk 1]
```diff
@@ -1,7 +1,12 @@
...
-  <calcite-input status="invalid" type="search" placeholder="Filter your files" value="adfo2h2"></calcite-input>
-  <calcite-input-message icon status="invalid"> Something doesn't look right </calcite-input-message>
+  Email address
+  <calcite-input
+    status="invalid"
+    type="email"
+    placeholder="Enter your email address"
+    value="invalidemailformat.com"
+  ></calcite-input>
+  <calcite-input-message icon status="invalid"> Enter a valid email address </calcite-input-message>
```
Comment (geospatialem) [inline packages/calcite-components/src/components/label/usage/Basic.md:10]: nitpick:
For readability, could the spaces at the beginning and tail end be removed?

File: packages/calcite-components/src/components/list/usage/Advanced.md [hunk 2]
```diff
@@ -1,37 +1,34 @@
 ```html
 <calcite-list>
-  <calcite-list-item label="Princess Bubblegum" description="Ruler of The Candy Kingdom">
+  <calcite-list-item
+    label="Dog"
+    description="(Canis familiaris) A carnivorous mammal closely related to the gray wolf that has been domesticated as a pet."
```
Comment (geospatialem) [inline packages/calcite-components/src/components/list/usage/Advanced.md:5]: For readability, WDYT about adding a sentence referencing the Latin name for this and other `list-item`'s? Maybe something like:

> Also known as Canis familiaris, a carnivorous...

File: packages/calcite-components/src/components/list/usage/Advanced.md [hunk 3]
```diff
@@ -1,37 +1,34 @@
...
-    ></calcite-avatar>
+    <calcite-icon scale="l" icon="layer" slot="content-start"></calcite-icon>
+    <calcite-avatar scale="l" slot="content-start" thumbnail="https://placekitten.com/g/300/300"></calcite-avatar>
     <calcite-icon scale="s" icon="check" slot="content-end" style="color: var(--calcite-ui-success)"></calcite-icon>
     <calcite-action icon="ellipsis" label="menu" slot="actions-end"></calcite-action>
     <calcite-action icon="x" label="remove" slot="actions-end"></calcite-action>
   </calcite-list-item>
-  <calcite-list-item label="Jake T. Dog" description="Part owner of the Tree House">
+  <calcite-list-item
+    label="Asteroid"
```
Comment (geospatialem) [inline packages/calcite-components/src/components/list/usage/Advanced.md:26]: Is this meant to be Asteroid, or possibly another animal? 🐻

File: packages/calcite-components/src/components/list/usage/Basic.md [hunk 4]
```diff
@@ -1,18 +1,16 @@
...
+    label="Meteoroid"
+    description="A small body of matter from outer space that enters Earth's atmosphere and appears like a streak of light."
   ></calcite-list-item>
   <calcite-list-item
-    label="Vestibulum commodo felis quis tortor.
-  "
-    description="Vestibulum auctor dapibus neque.
-  "
+    label="Asteroid"
+    description="A celestial body made of rock and metal that orbits the Sun."
```
Comment (geospatialem) [inline packages/calcite-components/src/components/list/usage/Basic.md:13]: ☄️

File: packages/calcite-components/src/components/list/usage/DragEnabledNested.md [hunk 5]
```diff
@@ -1,19 +1,24 @@
...
-        <calcite-list drag-enabled group="mygroup" selection-mode="multiple">
-          <calcite-list-item label="My Item 3" description="hello world">
-            <calcite-list drag-enabled group="mygroup" selection-mode="multiple"></calcite-list>
-          </calcite-list-item>
-          <calcite-list-item label="My Item 4" description="hello world"></calcite-list-item>
-        </calcite-list>
-      </calcite-list-item>
-      <calcite-list-item label="My Item 5" description="hello world"></calcite-list-item>
-    </calcite-list>
+<calcite-list drag-enabled group="Birds">
```
Comment (geospatialem) [inline packages/calcite-components/src/components/list/usage/DragEnabledNested.md:2]: To also showcase the `selectionMode`, WDYT about adding it into this usage example? Or creating a separate one to showcase the functionality?

File: packages/calcite-components/src/components/list/usage/DragEnabledNested.md [hunk 6]
```diff
@@ -1,19 +1,24 @@
...
+  <calcite-list-item open label="Eagles">
```
Comment (geospatialem) [inline packages/calcite-components/src/components/list/usage/DragEnabledNested.md:3]: 🦅

**Review by geospatialem — APPROVED** [review]
Awesome! 💪🏻 ✨ 

A few suggestions below for consideration prior to merging. WDYT?

File: packages/calcite-components/src/components/list/usage/Advanced.md (hunk 2 above)
Comment (DitwanP) [reply packages/calcite-components/src/components/list/usage/Advanced.md:5]: That makes sense to me, 👍

File: packages/calcite-components/src/components/list/usage/Advanced.md (hunk 3 above)
Comment (DitwanP) [reply packages/calcite-components/src/components/list/usage/Advanced.md:26]: Good catch!

File: packages/calcite-components/src/components/list/usage/DragEnabledNested.md (hunk 5 above)
Comment (DitwanP) [reply packages/calcite-components/src/components/list/usage/DragEnabledNested.md:2]: Adding a separate example for selectionMode.

File: packages/calcite-components/src/components/list/usage/MultipleSelections.md [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+      label="Baseball"
+      description="Baseball is a bat-and-ball sport played between two teams of nine players each, taking turns batting and fielding."
+    ></calcite-list-item>
+    <calcite-list-item
+      label="Golf"
+      description="Golf is a sport in which a small ball is struck with a club into a series of small holes in the ground, the object being to use the fewest possible strokes to complete the course."
+    ></calcite-list-item>
+  </calcite-list-item-group>
+</calcite-list>
+```
```
Comment (geospatialem) [inline packages/calcite-components/src/components/list/usage/MultipleSelections.md:18]: 🎾 ⚾ ⛳

**Review by jcfranco — APPROVED** [review]
Awesome improvements! 

Since the PR description will be included in the commit body when it's squash merged (by default), could you remove the redundant bits from the PR description before merging?

Other than that, this looks great! 

✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨
✨📚✨✨✨📚✨📚📚📚✨✨📚📚📚✨📚📚📚📚✨📚✨
✨📚📚✨✨📚✨✨📚✨✨📚✨✨✨✨📚✨✨✨✨📚✨
✨📚✨📚✨📚✨✨📚✨✨📚✨✨✨✨📚📚📚✨✨📚✨
✨📚✨✨📚📚✨✨📚✨✨📚✨✨✨✨📚✨✨✨✨✨✨
✨📚✨✨✨📚✨📚📚📚✨✨📚📚📚✨📚📚📚📚✨📚✨
✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨

**Pull request merged**